ceph-releases/jewel/daemon/check_zombie_mons.py

```diff
@@ -5,13 +5,20 @@
 import json
 
 MON_REGEX = r"^\d: ([0-9\.]*):\d+/\d* mon.([^ ]*)$"
-# kubctl_command = 'kubectl get pods --namespace=${CLUSTER} -l daemon=mon -o template --template="{ {{range .items}} \\"{{.metadata.name}}\\": \\"{{.status.podIP}}\\" ,   {{end}} }"'
+# kubctl_command = 'kubectl get pods --namespace=${CLUSTER} -l daemon=mon -o template --template="{ {{range .items}} \\"{{.metadata.name}}\\": \\"{{.status.podIP}}\\" ,   {{end}} }"'  # noqa E501
 if int(os.getenv('K8S_HOST_NETWORK', 0)) > 0:
-    kubectl_command = 'kubectl get pods --namespace=${CLUSTER} -l daemon=mon -o template --template="{ {{range  \$i, \$v  := .items}} {{ if \$i}} , {{ end }} \\"{{\$v.spec.nodeName}}\\": \\"{{\$v.status.podIP}}\\" {{end}} }"'
+    kubectl_command = 'kubectl get pods --namespace=${CLUSTER} ' \
+                        '-l daemon=mon ' \
+                        '-o template ' \
+                        '--template="{ {{range  \$i, \$v  := .items}} {{ if \$i}} , {{ end }} \\"{{\$v.spec.nodeName}}\\": \\"{{\$v.status.podIP}}\\" {{end}} }"'  # noqa E501
 else:
-    kubectl_command = 'kubectl get pods --namespace=${CLUSTER} -l daemon=mon -o template --template="{ {{range  \$i, \$v  := .items}} {{ if \$i}} , {{ end }} \\"{{\$v.metadata.name}}\\": \\"{{\$v.status.podIP}}\\" {{end}} }"'
+    kubectl_command = 'kubectl get pods --namespace=${CLUSTER} ' \
+                        '-l daemon=mon '\
```

Comment:
There is inconsistency in the number of spaces after a string and the backslash. The section above has a space between, but this does not. I you are making changes, maybe you can make sure there is a space between all these.